src/pool/srv_pool.c

```diff
@@ -925,14 +928,52 @@ handle_event(struct pool_svc *svc, struct pool_svc_event *event)
 		goto out_prop;
 	}
 
-	rc = pool_svc_exclude_rank(svc, event->psv_rank);
-	if (rc != 0) {
-		D_ERROR(DF_UUID": failed to exclude rank %u: "DF_RC"\n", DP_UUID(svc->ps_uuid),
-			event->psv_rank, DP_RC(rc));
-		goto out_prop;
-	}
+	if (DAOS_POOL_SUPPORT_REINT_ONLINE == 0)
+		goto DEAD;	
 
-	D_DEBUG(DB_MD, DF_UUID": excluded rank %u\n", DP_UUID(svc->ps_uuid), event->psv_rank);
+	if (event->psv_type == CRT_EVT_ALIVE) {
```

Comment:
DAOS_POOL_SUPPORT_REINT_ONLINE is to disable it by default?
may change to "if (event->psv_type == CRT_EVT_ALIVE && DAOS_POOL_SUPPORT_REINT_ONLINE)" so need not add the goto and "DEAD" label (and we commonly use lower-case letter for label).